tests/components/dynalite/test_light.py

```diff
@@ -15,8 +16,13 @@
     bridge = Mock()
     hass.data = {DOMAIN: {entry.entry_id: bridge}}
     await async_setup_entry(hass, entry, async_add)
-    bridge.register_add_entities.assert_called_once()
-    assert bridge.register_add_entities.mock_calls[0] == call(async_add)
+    bridge.register_add_devices.assert_called_once()
```

Comment:
In the light tests we just want to assert the state of the added light entity and that we can call the light services and that those calls call into the correct dynalite api. Patch the dynalite library methods and assert the calls of those patched methods.

Set up the dynalite component, which will load the light platform, which will add an entity. Then do the assertions.

Reply:
fixed